kubernetes/kubernetes.go

```diff
@@ -120,8 +138,9 @@ func CollectKubeMetrics(config KubeAgentConfig) {
 	err := fetchDiagnostics(kubeAgent.Clientset, config.Namespace, kubeAgent.msExportDirectory)
 
 	if err != nil {
+		log.Warnf(rbacError, kbProvisionURL)
 		log.Warnf("Warning non-fatal error: Agent error occurred retrieving runtime diagnostics: %s ", err)
-		log.Warnf("For more information see: %v", kbURL)
+		log.Warnf("For more information see: %v", kbErrorURL)
```

Comment:
⭐ 
The naming for this var:`kbErrorURL` can cause confusion. Originally I thought this meant it was the URL to the error occurring but after looking at the link, it just goes to our help docs. What do you think about updating the naming to something like `kbTroubleshootDocsURL` or `kbHelpDocsURL`.

Reply:
Yeah it goes to our Kubernetes documents where our typical errors are provided for customer viewing.  That's why I named it `kbErrorURL` because we know that's where the public error information is rather than the provisioning information.